src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,117 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exeptions.UserValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService regService;
+
+    @BeforeEach
+    void setUp() {
+        regService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_UserIsNull_NotOk() {
```

Comment:
After the underscore, the word should be started in lowercase. Fix it in all tests
Example:
![image](https://user-images.githubusercontent.com/32339836/214036878-6f9be2fa-bed0-46f9-ae9f-da7c6b83a889.png)